yang-module-versioning/draft-ietf-netmod-yang-module-versioning.xml

```diff
@@ -772,27 +764,39 @@ module ietf-yang-revisions {
   // RFC Ed.: replace XXXX (inc above) with actual RFC number and
   // remove this note.
 
-  revision 2022-10-10 {
+  revision 2022-11-21 {
     rev:revision-label 1.0.0-draft-ietf-netmod-yang-module-versioning-07;
     description
       "Initial version.";
     reference
       "XXXX: Updated YANG Module Revision Handling";
   }
 
+  typedef revision-date {
+    type string {
+      pattern '[0-9]{4}-(1[0-2]|0[1-9])-(0[1-9]|[1-2][0-9]|3[0-1])';
+    }
+    description
+      "A date associated with a YANG revision.
+
+      Matches dates formatted as YYYY-MM-DD.";
+    reference
+      "RFC 3339: Date and Time on the Internet: Timestamps";
```

Comment:
I think we may want to reference RFC 7950 instead. We're really following whatever RFC7950 did (if it matches some other RFC, that's a coincidence or up to 7950).

Reply:
Yes, okay.